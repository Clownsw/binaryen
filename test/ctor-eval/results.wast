(module
  (func "test1" (result i32)
    ;; The presence of a result stops us from evalling this function (at least
    ;; for now).
    (i32.const 42)
  )
)
