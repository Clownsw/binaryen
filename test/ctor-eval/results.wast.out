(module
 (type $none_=>_i32 (func (result i32)))
 (export "test1" (func $0))
 (func $0 (result i32)
  (i32.const 42)
 )
)
